css/about.css

```diff
@@ -0,0 +1,79 @@
+.about-second-section .wlit-logo {
+  border: 1px solid black;
+}
+
+.about-second-section {
+  border-bottom: 1px solid rgb(223, 223, 223);
+}
+
+.first-content .detail-desc {
+  border: 1px solid rgb(223, 223, 223);
+  background: rgb(255, 255, 255);
+  width: 80%;
+}
+
+.events h4::after {
+  content: "";
+  display: block;
+  border: 1px solid #ec5242;
+  width: 5%;
+  margin: 10px auto;
+}
+
+.first-content {
+  background-image: url('../images/mobile_intro_bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+}
+
+.events .events-images .first-event {
+  height: 30%;
+  width: 50%;
+  margin-right: 10px;
+}
+
+.events .events-images .second-event {
+  height: 30%;
+  width: 50%;
+}
+
+.up-footer {
+  display: none;
+}
+
+@media screen and (max-width: 767.99px) {
+  .first-content {
+    width: 100%;
+  }
+
+  .first-content .content-img img {
+    width: 100%;
+    height: auto;
+  }
+
+  .about-second-section .wlit-logo {
+    width: 100%;
+    height: auto;
+  }
+
+  .events .events-images {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .events .events-images .first-event {
+    height: 50%;
+    width: 90%;
+    margin-bottom: 20px;
+  }
+
+  .events .events-images .second-event {
+    height: 30%;
+    width: 90%;
+  }
+
+  .up-footer {
+    display: flex;
+  }
```

Comment:
- How about we do a little improvement in this section. The implementation and the design slightly don't match.

| The Provide Design |
| --- |
| ![partner-design](https://user-images.githubusercontent.com/45002654/133671844-274587b4-c26f-44f1-ab75-0802bec0d2d4.PNG) |

| Your implementation |
| --- |
| ![partner-imp](https://user-images.githubusercontent.com/45002654/133671905-cc126c16-e6f3-4942-82f1-03518374f6cb.PNG) |

## What to improve in this section
- The background-color
- Cente- align the Partner logos and reduce the margins between them.